js/i18n.js

```diff
@@ -456,12 +597,13 @@ function appendFilter(filterExpression, rule) {
  *                                                where each object contains returnedFields
  * @return boolean - true if search initiated, if false then search error
  */
-function searchManifest(matchLimit, languages, user_name, repo_name, resource, full_text, returnedFields, onFinished) {
+function searchManifest(matchLimit, languages, user_name, repo_name, resID, resType, title, time, manifest, full_text, returnedFields, onFinished) {
     try {
         var tableName = getManifestTable();
         var expressionAttributeValues = {};
         var filterExpression = "";
-        var expressionAttributeNames = {  };
+        var expressionAttributeNames = {};
+        var projectionExpression = null;
 
         if (languages) {
             var languageStr = "[" + languages.join(",") + "]"; // convert array to set string
```

Comment:
While not in your commit, a search of language codes should be an exact match as that is what the user specifically chose from a list of languages.